modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-unsafe-recover.adoc

```diff
@@ -0,0 +1,41 @@
+= rpk cluster partitions unsafe-recover
+
+Recover unsafely from partitions that have lost majority.
+
+IMPORTANT: This operation is unsafe because it allows the forced leader election of the partitions that have lost majority when nodes are gone and irrecoverable; this may result in data loss.
+
+This command allows you to unsafely recover all data adversely affected by the loss of the nodes specified in the `--from-nodes` flag. 
+
+You can perform a dry run and verify the partitions that will be recovered by using the `--dry` flag.
+
+== Usage
+
+[,bash]
+----
+rpk cluster partitions unsafe-recover [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--dry |- |Dry run: print the partition movement plan. Does not execute it.
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|--from-nodes |- |ints   Comma-separated list of node IDs from which to recover the partitions.
+
+|-h, --help |- |Help for unsafe-recover.
+
+|--no-confirm |- |Disable confirmation prompt.
+
+|--config |string |Redpanda or rpk config file; default search paths are ~/.config/rpk/rpk.yaml, $PWD, and /etc/redpanda/`redpanda.yaml`.
```

Comment:
Looks like formatting error here, and in the -X details below

Reply:
Needs to be solved globally. Will fill another PR later for that.